components/Competition/Criteria/Mitra/index.vue

```diff
@@ -73,7 +73,7 @@ export default {
   },
   methods: {
     onShowDetailContent () {
-      // @todo: to show info detail content
+      this.$emit('onClick', true)
```

Comment:
minor:

hello @doohanas , according to official Vue.js docs, we should use `kebab-case` naming convention for custom events

ref: https://eslint.vuejs.org/rules/custom-event-name-casing.html

Reply:
Hello kak @Ibwedagama this changes for custom event naming in [here](https://github.com/jabardigitalservice/dedi-landing-page/pull/129/commits/908af654335cb9996de73867fd87912b3ba25817)